src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {observer, inject } from "mobx-react";
+import {reaction, computed} from "mobx";
+import validateParameters from 'shared/lib/validateParameters';
+import LoadingIndicator from "shared/components/loadingIndicator/LoadingIndicator";
+import ValidationAlert from "shared/components/ValidationAlert";
+import ResultsViewMutationTable from "./mutation/ResultsViewMutationTable";
+import {ResultsViewPageStore} from "./ResultsViewPageStore";
+
+const resultsViewPageStore = new ResultsViewPageStore();
+
+(window as any).resultsViewPageStore = resultsViewPageStore;
+
+export interface IResultsViewPageProps {
+    routing: any;
+}
+
+@inject('routing')
+@observer
+export default class ResultsViewPage extends React.Component<IResultsViewPageProps, {}> {
+
+    constructor(props: IResultsViewPageProps) {
+        super();
+
+        const reaction1 = reaction(
+            () => props.routing.location.query,
+            query => {
+
+                const validationResult = validateParameters(query, ['studyId']);
```

Comment:
I didn't include the full list of parameters, and validation here yet. Parameter names and structure, as well as the route is subject to change